working-with-openai/setting-up-open-ai-postman.md

```diff
@@ -0,0 +1,91 @@
+# Setting up the OpenAI API in Postman
```

Comment:
It might make it a little easier for students to find information if we differentiate the lesson names, especially since longer names can be truncated in the side bar. A couple potential ideas:

```suggestion
# Calling the OpenAI API from Postman
```

```suggestion
# Sending requests to the OpenAI API from Postman
```